versioned_plugins.rb

```diff
@@ -310,7 +310,7 @@ def extract_doc(doc, plugin_full_name, release_tag, release_date, changelog_url)
 
     if repair?
       content = content.gsub(/^====== /, "===== ")
-        .gsub("[source]", "[source,shell]")
+        .gsub(/^\[source\]$/, "[source,shell]")
```

Comment:
When Ruby's `String#gsub` gets a string as its first argument, it matches the given substring _anywhere_.

By supplying a regular expression, we can be more specific about what we need matched.

In this case the regular expression `^\[source\]$` above matches the sequence:
 - `^`: the beginning of a line,
 - `\[`: an open square-bracket character,
 - `s`,
 - `o`,
 - `u`,
 - `r`,
 - `c`,
 - `e`,
 - `\]`: a closing quare-bracket
 - `$`: the end of a line